tests/code_wars/test_split_strings.py

```diff
@@ -0,0 +1,19 @@
+from code_challenges.code_wars.split_strings import split_the_string
+
+
+def test_split_the_string_even_string_len():
+    actual = split_the_string("ijsnytfg")
+    expected = ["ij", "sn", "yt", "fg"]
+
+    assert (
+        actual == expected
+    ), "you should renturn a list with the strings evenly split into pairs."
```

Comment:
nit: typo  renturn --> return

----
nit is short for nitpick and suggests this is very minor but the review cares about such things.